server/src/tasks/deployment/email.js

```diff
@@ -0,0 +1,17 @@
+const sendgridUtils = require('@utils/sendgrid')
+
+const email = async (networks, communityProgress) => {
```

Comment:
I though of tasks about ethereum transaction that take time, but we can put the email handling here also